resources/eventing/README.md

```diff
@@ -4,11 +4,16 @@ This Helm chart contains all components required for eventing in Kyma:
 
 Components:
 - event-publisher-proxy
+- controller
 
 ## Event Publisher Proxy
 
 This component receives Cloud Event publishing requests from the cluster workloads (microservice or Serverless functions) and redirects them to the Enterprise Messaging Service Cloud Event Gateway. Click [here](https://github.com/kyma-project/kyma/tree/master/components/event-publisher-proxy) for more details.
 
+## Controller
+
+This component manages the internal infrastructure in order to receive an event for all subscriptions
+
 ## Install
 
```

Comment:
Let's also add a short line how to install the CRD then installed via helm only:

```
kubectl apply -f resources/cluster-essentials/files/subscriptions.eventing.kyma-project.io.crd.yaml
```

Reply:
Done